src/khepri_machine.erl

```diff
@@ -1253,6 +1253,12 @@ handle_aux(
                Projection <- Projections]
       end),
     {no_reply, AuxState, LogState};
+handle_aux(
```

Comment:
why not make use of the screen estate and put some of the arguments on this line? with this formatting the function head takes up 3 lines when it could just use 2 (and maintain left to right reading)

Reply:
I don't have an opinion either way, this was just to fit the existing indentation in the project

Reply:
Yeah, that’s me who is not a fan of variable indentation where you end up with:
```erlang
function(Arg1,
         Arg2)
```

If all arguments can’t fit on one line, I put all of them on the next line:
```erlang
function(
  Arg1,
  Arg2)
```

By "variable indentation", I mean that the column on which the arguments start would depend on the length of the function name. In almost all cases, they would start more on the right than the function body:
```erlang
function(Arg1,
         Arg2) ->
    do().
```
Instead of keeping some kind of hierarchy where the function body is more indented than the function clause:
```erlang
function(
  Arg1,
  Arg2) ->
    do().
```